docs/tasks/debug-application-cluster/run-command-container-kubectl-exec.md

```diff
@@ -0,0 +1,131 @@
+---
+assignees:
+- caesarxuchao
+- mikedanese
+title: Running Commands in a Container using kubectl exec
+---
+
+{% capture overview %}
+
+This page shows how to use `kubectl exec` to run commands in a Container.
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+## Using kubectl exec to view the environment variables of a Container
```

Comment:
Getting a shell is a much more common operation, maybe we should say that instead?